e2e-tests/insurance/cypress/e2e/journeys/your-business/company-details/company-details-save-and-back.spec.js

```diff
@@ -59,28 +58,8 @@ describe('Insurance - Your business - Company details page - Save and go back',
     it('should not display validation errors and redirect to task list with status of "In progress"', () => {
       cy.navigateToUrl(url);
 
-      companyDetails.tradingNameYesRadioInput().click();
-      companyDetails.tradingAddressYesRadioInput().click();
-      saveAndBackButton().click();
-
-      cy.assertUrl(`${Cypress.config('baseUrl')}${ROUTES.INSURANCE.ROOT}/${referenceNumber}${ALL_SECTIONS}`);
-      cy.checkTaskStatus(task, IN_PROGRESS);
-    });
-
-    it('should render valid submitted values when going back to the page', () => {
-      cy.navigateToUrl(url);
-
-      companyDetails.tradingNameYesRadioInput().should('be.checked');
-      companyDetails.tradingAddressYesRadioInput().should('be.checked');
-    });
-  });
-
-  describe(`when required fields are completed and ${INPUT} is entered incorrectly`, () => {
-    it('should not display validation errors and redirect to task list', () => {
-      cy.navigateToUrl(url);
-
-      cy.completeCompanyDetailsForm({ companiesHouseNumber: '**/*' });
-
+      companyDetails[TRADING_NAME].yesRadioInput().click();
+      companyDetails[TRADING_ADDRESS].yesRadioInput().click();
```

Comment:
Could we use `yesRadioInput` from `pages/shared` instead and delete this from `companyDetails`? I think we have a tech ticket to do this

Reply:
done